templates/java/mobile/lombok.config

```diff
@@ -0,0 +1,8 @@
+lombok.addGeneratedAnnotation = false
```

Comment:
As said in previous comments this should be optional.

Reply:
This file only is needed if we add lombok dependency, if not it can be added in the filesToRemove option from config json.